micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/SidebarAdd.js

```
@@ -0,0 +1,571 @@
+        if (field.name === "id") return null;
+        if (disabled && (field.type === "multiselect" || field.type === "switch")) return null;
+        return (
+            <div key={field.name} style={{
+                display: "grid",
+                gridTemplateColumns: "200px 1fr",
+                alignItems: "center",
+                gap: "1rem",
+                marginBottom: "1rem"
+            }}>
+                <div style={{ display: "flex", gap: "0.5rem" }}>
+                    <label style={{ fontWeight: "500" }}>{t(field.label || field.name.toUpperCase())}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)